.travis/script.sh

```diff
@@ -450,6 +450,46 @@ build_samr21() {
     ./tests/toranj/start.sh || die
 }
 
+[ $BUILD_TARGET != size-report ] || {
+    [ ${TRAVIS_PULL_REQUEST} != false ] || die
+
+    export PATH=/tmp/gcc-arm-none-eabi-7-2018-q2-update/bin:$PATH || die
+
+    mkdir ../output
+
+    # pull request
+    OPENTHREAD_FLAGS="BORDER_AGENT=1 BORDER_ROUTER=1 CHANNEL_MANAGER=1 CHANNEL_MONITOR=1 CHILD_SUPERVISION=1 COAP=1 COAPS=1 COMMISSIONER=1 DHCP6_CLIENT=1 DHCP6_SERVER=1 DIAGNOSTIC=1 DISABLE_DOC=1 DNS_CLIENT=1 ECDSA=1 FULL_LOGS=1 JAM_DETECTION=1 JOINER=1 LINK_RAW=1 MAC_FILTER=1 MTD_NETDIAG=1 SERVICE=1 SLAAC=1 SNTP_CLIENT=1 TIME_SYNC=1 UDP_FORWARD=1"
+
+    git checkout -- . || die
+    git clean -xfd || die
+    ./bootstrap || die
+    make -f examples/Makefile-nrf52840 ${OPENTHREAD_FLAGS} || die
+    mv output/nrf52840 ../output/nrf52840-b
+
+    git checkout ${TRAVIS_BRANCH}
+    git submodule update --init
```

Comment:
Just curios:  How are we getting to master branch for comparison? 
Is it somehow derived from the PR branch itself (or is it top of master?)

Reply:
${TRAVIS_BRANCH} will be master for pull requests. We just switch to master branch here.

Reply:
Perfect. Thanks.

Reply:
Is `${TRAVIS_BRANCH}` the current HEAD of the target branch? Or the common ancestor (i.e. git-merge-base)? I think the common ancestor is what we want.

Reply:
Is HEAD of master better? I think we want to know the size diff before and after the merge, right?

Reply:
@jwhui updated. https://github.com/openthread/size-report/pull/1